lib/utils.js

```diff
@@ -26,10 +27,24 @@ function doloadGPTScript(resolve, reject, limitedAds) {
   };
 
   document.getElementsByTagName('head')[0].appendChild(scriptTag);
+
+  if (timerId !== null) {
+    clearTimeout(timerId);
+  }
 }
 
 function loadGPTScript() {
   var limitedAds = arguments.length > 0 && arguments[0] !== undefined ? arguments[0] : false;
+  var deferAds = arguments.length > 1 && arguments[1] !== undefined ? arguments[1] : true;
```

Comment:
```suggestion
  var limitedAds = arguments.length > 0 && arguments[0] ? arguments[0] : false;
  var deferAds = arguments.length > 1 && arguments[1] ? arguments[1] : true;
```

Having the field directly in the condition should be enough if we are checking if the value exists.

If we want to strictly check for `undefined`, we need to make use of `typeof`. The way it is written in the library is an anti pattern.